internal/network/command_dns_record_create.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	privateLinkAccessPoint = "PrivateLinkAccessPoint"
+	privateLinkAccessPointKind = "PrivateLinkAccessPoint"
```

Comment:
This only gets used in one place, so I would consider hardcoding it there to avoid needing to look up what this constant is equal to.